frontend/src/scenes/session-recordings/SessionRecordingsPlaylist.tsx

```diff
@@ -33,28 +33,43 @@ const SessionRecordingPlaylistItem = ({
     isActive: boolean
     onClick: () => void
 }): JSX.Element => {
-    const iconClassnames = isActive ? 'text-lg text-muted-alt' : 'text-lg text-muted-alt opacity-75'
+    const { setPropertyFilters } = useActions(sessionRecordingsListLogic)
+    const { propertyFilters } = useValues(sessionRecordingsListLogic)
```

Comment:
I'd be tempted to pull this up to the SessionRecordingsPlaylist (`onPropertyClick`) rather than nesting the logic here as it could get a bit messy. 

A lot of the logic usage in SR is a bit messy but for me this feels like a more "pure" component that should let the parent choose what to do (we might have this list in a different view where clicking the icons doesn't do anything for example)

Reply:
Good call